files/familysearch-javascript-sdk/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "familysearch-javascript-sdk",
+  "repo": "FamilySearch/familysearch-javascript-sdk",
+  "files": {
+    "include": ["*.js"]
```

Comment:
@jimaek Correct. That release was done incorrectly. In all other tags, such as the latest [1.0.0-beta3](https://github.com/FamilySearch/familysearch-javascript-sdk/tree/1.0.0-beta3) the files do exist.